frontend/src/components/communication/communicationList.tsx

```diff
@@ -7,10 +7,17 @@ import { ICommunication } from "../../api/entities/CommunicationEntity";
 import { getAllCommunicationFromPage } from "../../api/calls/communicationCalls";
 import CommunicationListItem from "./communicationListItem";
 
+/**
+ * Properties needed for [CommunicationList].
+ */
 export interface CommunicationListProps {
     student: IStudent | undefined;
 }
 
+/**
+ * Component listing all communications of a given student. (In order as provided by the list)
```

Comment:
```suggestion
 * Component listing all communication with a given student. (In order as provided by the list)
```

Can the order of this list be set?